src/FSAL/FSAL_CORTXFS/mds.c

```diff
@@ -1133,13 +1151,16 @@ kvsfs_layoutcommit(struct fsal_obj_handle *obj_hdl,
 			     struct kvsfs_fsal_obj_handle,
 			     obj_handle);
 
-	/* @todo: To supress compile warnings, copying data from
-	 *	  struct cfs_fh to struct kvsfs_file_handle for now,
-	 *	  but these two structures are very much apart in
-	 *	  implementation and needs revisit.
+	/* @todo for pNFS : To convert host file handle to wire,
+	 * one must use relevant API's that can construct
+	 * struct kvsfs_file_handle from given struct cfs_fh.
+	 * (myself->handle) should be converted to kvsfs_ds_handle.
```

Comment:
Pls correct this comment. kvsfs_ds_handle needs to be replaced by opaque file handle in future.

Reply:
Done